src/borg/testsuite/repository.py

```diff
@@ -544,17 +547,17 @@ def test_shadowed_entries_are_preserved(repository):
         # if it's not sparse enough (symbolized by H(2) here).
         repository.delete(H(1))
         repository.put(H(2), fchunk(b"1"))
-        delete_segment = get_latest_segment()
+        del_segment = get_latest_segment()
```

Comment:
delete_segment shadowed a helper function name from outer scope. Renamed for clarity.